WebDAVClient/Helpers/WebDAVException.cs

```diff
@@ -1,47 +1,63 @@
 using System;
+using System.Runtime.InteropServices;
 using System.Runtime.Serialization;
 using System.Web;
 
 namespace WebDAVClient.Helpers
 {
-    public class WebDAVException : HttpException
+    public class WebDAVException : Exception
```

Comment:
System.Web doesn't have a `HttpException` anymore, therefore I needed to make my own.
If you have any recommendations, then tell me please.